src/main/resources/game_publishers.csv

```diff
@@ -0,0 +1,1208 @@
+Publisher, Count
+Electronic Arts, 188
+Ubisoft, 133
+Activision, 123
+Sega, 83
+Eidos Interactive, 80
+THQ, 71
+EA Sports, 71
+, 69
```

Comment:
NoName publisher?

Reply:
:D jojo toho jsem si taky všimla... - asi bude lepší ho tam vůbec nevyhazovat, že ?

Reply:
Nedával bych ho tam, protože to může být třeba způsobený špatným parsováním.

Reply:
to jsem si taky myslela, ale ono reálně tam některé ty hry toho publishera prostě nemaj. Ale hodila jsem to do podmínky, že když je to prázdný string tak se to tam neuloží ;).